reactive/kotlinx-coroutines-reactor/src/Flux.kt

```diff
@@ -32,9 +42,238 @@ import kotlin.coroutines.*
  * **Note: This is an experimental api.** Behaviour of publishers that work as children in a parent scope with respect
  *        to cancellation and error handling may change in the future.
  */
+
 @ExperimentalCoroutinesApi
 fun <T> CoroutineScope.flux(
     context: CoroutineContext = EmptyCoroutineContext,
     @BuilderInference block: suspend ProducerScope<T>.() -> Unit
-): Flux<T> =
-    Flux.from(publish(newCoroutineContext(context), block = block))
+): Flux<T> = Flux.from(reactorPublish(newCoroutineContext(context), block = block))
+
+@ExperimentalCoroutinesApi
+public fun <T> CoroutineScope.reactorPublish(
```

Comment:
Why do we have to introduce public `reactorPublish` API?
Integration should work with `flux` builder.